capa/features/insn.py

```diff
@@ -29,6 +29,10 @@ def get_value_str(self):
         return capa.render.utils.hex(self.value)
 
 
+# max recognized structure size (and therefore, offset size)
+MAX_STRUCTURE_SIZE = 0x10000
```

Comment:
I find this pretty big. How did you pick this value?

Reply:
i wanted a round number large than structures/arrays i commonly work with. since 0x1000 is `PAGE_SIZE`, the next choice was 0x10000. happy to change if necessary.

Reply:
ok, makes sense and I'm fine keeping it as is